src/osm_configurator/model/project/configuration/category_manager.py

```diff
@@ -37,7 +37,7 @@ def get_activated_attribute(self) -> List[Attribute]:
                     _activated_attributes.append(attribute)
         return _activated_attributes
 
-    def get_category(self, index):
+    def get_category(self, index: int) -> Category:
```

Comment:
I dont think this is a useful method, a better method would be get_category_by_name, where you get the category by its name not my its index, this is already implemented in model branch.
So you might want to remove this method if you dont personally need it.

Reply:
agree